pittAPI.py

```diff
@@ -375,29 +375,49 @@ class PeopleAPI:
     def __init__(self):
         pass
 
-    def get_person(self, query):
+    def get_person(self, query, maxPeople):
         '''
-        Doesn't work completely for now.
+        Doesn't work completely for now. IT WORKS
         Returns a dict with URLs of user profiles. No scraping yet.
         '''
 
         query = query.replace(' ', '+')
-
-        cmd = """
-        curl -k -s "https://136.142.34.69/people/search?search=Search&filter={}&_region=kgoui_Rcontent_I0_Rcontent_I0_Ritems"
-        """.format(query)
-
-        response = subprocess.check_output(cmd, shell=True)
-
-        results = []
-        while("formatted" in response):
-            response = response[response.index('"formatted"'):]
-            response = response[response.index(":") + 2:]
-            response_str = response[:response.index('}') - 1]
-            response_str = response_str.replace('\u0026', '&')
-            response_str = response_str.replace('\\', '')
-            if '&start=' not in response_str:
-                results.append("https://136.142.34.69" + response_str)
-            response = response[response.index('}') :]
-
-        return results
+        persons_list = []
+        tens = 0
+        while tens < maxPeople:
+            url = "https://136.142.34.69/people/search?search=Search&filter={}&_region=kgoui_Rcontent_I0_Rcontent_I0_Ritems"
+            url += '&_region_index_offset=' + str(tens) + '&feed=directory&start=' + str(tens)
+            cmd = 'curl -k -s ' + '"'  + url + '"'
+
+
+            cmd = cmd.format(query)
+            response = subprocess.check_output(cmd, shell=True)
+            if not response: #no more responses, so break
+                break
+            #print(response)
```

Comment:
Wait, remove the comment? The if + break statement would be useful if the user gives a max of  50 but there's only like 2 results